runtime/lua/vim/filetype.lua

```diff
@@ -52,8 +55,12 @@ local extension = {
   art = 'art',
   asciidoc = 'asciidoc',
   adoc = 'asciidoc',
+  asa = vim.g.filetype_asa or 'aspvbs',
```

Comment:
This won’t work, it needs to be in a function

Reply:
True, sorry for the wrong suggestion